examples/counter/src/state.rs

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use serde::{Deserialize, Serialize};
+use linera_sdk::views::{linera_views, RegisterView, RootView, ViewStorageContext};
 
 /// The application state.
-#[derive(Clone, Copy, Debug, Default, Deserialize, Serialize)]
+#[derive(RootView)]
+#[view(context = "ViewStorageContext")]
 pub struct Counter {
-    pub value: u64,
+    pub value: RegisterView<u64>,
```

Comment:
IMHO I think no, because the goal of this example is to be the simplest possible. Previously it was simpler to use `SimpleStateStorage`, now it's simpler to just derive `RootView`.

Reply:
Sounds good